Dockerfile.template

```diff
@@ -101,12 +101,8 @@ ENV COMPOSER_ALLOW_SUPERUSER 1
 ENV COMPOSER_MEMORY_LIMIT -1
 COPY --from=composer:latest /usr/bin/composer /usr/bin/composer
 
-RUN set -eux; \
-    composer global require "hirak/prestissimo:^0.3" --prefer-dist --no-progress --no-suggest --classmap-authoritative; \
-    composer clear-cache
 ENV PATH="${PATH}:/root/.composer/vendor/bin"
```

Comment:
I think we can remove this too, as it should only be needed when there are global packages which is not the case anymore.
```suggestion
```